modules/ingest-geoip/src/test/java/org/elasticsearch/ingest/geoip/MaxMindSupportTests.java

```diff
@@ -472,7 +492,7 @@ public void testUnknownMaxMindResponseClassess() {
      * This tests that this test has a mapping in TYPE_TO_MAX_MIND_CLASS for all MaxMind classes exposed through IpDatabase.
      */
     public void testUsedMaxMindResponseClassesAreAccountedFor() {
-        Set<Class<? extends AbstractResponse>> usedMaxMindResponseClasses = getUsedMaxMindResponseClasses();
+        Set<Class<? extends AbstractResponse>> usedMaxMindResponseClasses = MAX_MIND_CLASS_TO_IMPLEMENTATION_CLASS.keySet();
```

Comment:
This method is now pointless -- it's just comparing two sets that we define in the test. Ideally, we could get the set of maxmind responses that we use from IpDataLookupFactory. I wouldn't consider it a blocker for this PR, but if we don't do something like that we might as well delete this method so we don't get a false sense of security from it.

Reply:
Fair, I dropped the test in a5d01f94025dd0fa4b8fa85bb26cb5dab311473b, and I think you and I should take as a task that it would be nice to do _something_ here (but not necessarily on this PR).